cli/args.go

```diff
@@ -257,6 +264,21 @@ func parseStringArg(args []string, argName string, defaultValue string) (string,
 	return defaultValue, nil
 }
 
+// Find a int argument (e.g. --foo 1) of the given name in the given list of arguments. If it's present,
+// return its value. If it is present, but has no value, return an error. If it isn't present, return defaultValue.
+func parseIntArg(args []string, argName string, defaultValue string) (int, error) {
+	for i, arg := range args {
+		if arg == fmt.Sprintf("--%i", argName) {
```

Comment:
What is `%i`? Aren't strings usually `%s`?

Reply:
completely right, i tested with a num, fixed